ui/plugin-lifecycle/src/main/pipes/capitalizefirst/capitalizefirst.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from "@angular/core";
+/*
+ * Capitalize the first letter of the string
+ * Takes a string as a value.
+ * Usage:
+ *  value | capitalizefirst
```

Comment:
sic: capitalizeFirst

Reply:
The term is renamed in last commit.

Reply:
The term is renamed in "Create capitalize pipe" commit.